src/qt/zc2sigmapage.cpp

```diff
@@ -114,3 +136,27 @@ void Zc2SigmaPage::selectionChanged() {
     }
     ui->remintButton->setDisabled(!enabled);
 }
+
+void Zc2SigmaPage::updateAvailableRemints() {
+    model->updateRows();
+}
+
+bool Zc2SigmaPage::showZc2SigmaPage() {
+    {
+        LOCK(cs_main);
+        if(!znodeSync.IsBlockchainSynced()) {
+            return false;
+        }
+    }
+
+    {
+        LOCK(cs_main);
+        const Consensus::Params &params = Params().GetConsensus();
+        if (chainActive.Height() < params.nSigmaStartBlock ||
```

Comment:
Will be better to use `sigma::IsSigmaAllowed` instead to centralize logic also it more readable too.